### DISCUSSION TITLE
PHPstorm installation ot possible anymore

### DISCUSSION THREAD

**Discussion by saschaende** [community/community#41172] [Copilot Conversations]
### Select Topic Area

Product Feedback

### Body

PHPstorm installation ot possible anymore. Paying for....?

![image](https://user-images.githubusercontent.com/17617037/206667129-3e21c351-3f4c-4f3b-bcbe-00b6fc877acf.png)

**Comment by iamcjwilliams** [comment]
@saschaende Sorry that you're facing trouble using Copilot with PHPStorm. Please uninstall and then reinstall the plugin, allow it some time to connect, and see if that helps. Also, make sure that you are running the latest version of Copilot
